src/tests/fixtures/rendering.html

```diff
@@ -5,6 +5,22 @@
     <title>Turbo</title>
     <script src="/dist/turbo.es2017-umd.js" data-turbo-track="reload"></script>
     <script src="/src/tests/fixtures/test.js"></script>
+    <script type="module">
+      const permanentAudioElementLoaded = new Promise(resolve => {
+        addEventListener("DOMContentLoaded", () => {
+          const element = document.getElementById("permanent-audio")
+          element.addEventListener("canplaythrough", () => resolve(element), { once: true })
```

Comment:
Could this be declared as `addEventListener("canplaythrough", () => ... { once: true, capture: true })` so that we're not declaring nested handlers?